website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -152,3 +152,15 @@ When you trigger a run through the command line, Terraform Cloud applies the fol
 - **C_Variable_Set:** `KEY2`. For this run, `KEY2` equals `c`. This is a global variable set with no overwritten values.
 
 ![An example scenario demonstrating variable precedence in Terraform Cloud](/img/docs/variable-precedence-example.png)
+
+
+## Precedence with Priority Variable Sets
```

Comment:
@ritsok & @shernandez5 I tried combining both of your suggestions and rewrote this section to this: